logic/src/commonMain/kotlin/com/wire/kalium/logic/data/id/IdMappers.kt

```diff
@@ -0,0 +1,20 @@
+package com.wire.kalium.logic.data.id
+
+import com.wire.kalium.network.api.base.model.UserAssetDTO
+import com.wire.kalium.persistence.dao.QualifiedIDEntity
+
+internal typealias NetworkQualifiedId = com.wire.kalium.network.api.base.model.QualifiedID
+internal typealias PersistenceQualifiedId = QualifiedIDEntity
+
+// QualifiedID
+fun QualifiedID.toApi(): NetworkQualifiedId = NetworkQualifiedId(value, domain)
+fun QualifiedID.toDao(): PersistenceQualifiedId = PersistenceQualifiedId(value, domain)
+
+fun QualifiedIDEntity.toModel(): QualifiedID = QualifiedID(value, domain)
+fun QualifiedIDEntity.toApi(): NetworkQualifiedId = NetworkQualifiedId(value, domain)
+
+fun NetworkQualifiedId.toModel(): QualifiedID = QualifiedID(value, domain)
+fun NetworkQualifiedId.toDao(): PersistenceQualifiedId = PersistenceQualifiedId(value, domain)
+
+fun UserAssetDTO.toDao(domain: String): QualifiedIDEntity = PersistenceQualifiedId(key, domain)
+fun UserAssetDTO.toModel(domain: String): QualifiedID = QualifiedID(key, domain)
```

Comment:
`internal` modifier.

All mapping stuff shouldn't really be exposed to consumer apps.

We should actually go a bit deeper in a future PR and make sure that API models, DB models, etc. also shouldn't be exposed.

Reply:
Yup I agree also I think in future we should remove API and DB models from use cases